### PR TITLE
feat(website): Added the new community header / footer

### PR DESCRIPTION
**Pull request by LukyVj**
This is more or less how it looks now ( Since the background is using canvas and have a lot of interactions, my software to capture the entire page kinda bug, but you can have the big picture ) 

| Before  | After |
| ------------- | ------------- |
|  |   |

### CODE REVIEW

**Comment by vvo** [comment]
Thanks!

Can you rename it to "docs(style)...". A feat() would be a new feature of the library, this is a documentation change.

Also has conflicts.

**Comment by LukyVj** [comment]
@vvo Just fixed the conficts :) 
About to rename the commit!

File: docs/_layouts/default.html [hunk 1]
```diff
@@ -4,7 +4,7 @@
     <meta charset="utf-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1">
-
+    <!-- <meta content="width=device-width,initial-scale=1.0,minimum-scale=1.0,maximum-scale=1.0,user-scalable=no" name=viewport> -->
```
Comment (LukyVj) [inline docs/_layouts/default.html]: Let me remove this line first.

**Comment by vvo** [comment]
Sass linting failed (npm run lint:css). I am no CSS expert and I think we don't have a single linting ruleset for SCSS.

So either you comply with it, roll your own linting style/tooling or remove the SCSS linting. Up to you here :) But it has to pass

**Comment by LukyVj** [comment]
Thanks. 
So it's mostly yelling because Hexa decimal must be in uppercase. 
Fixin it now :)

**Pull request merged**